tests/test_output_handling.py

```diff
@@ -71,21 +74,59 @@ def test_invert_uv_inplace():
     assert np.allclose(data[..., 1], wd)
 
 
-def test_forward_pass_collection():
-    """Test forward pass collection."""
+def test_h5_out_and_collect():
```

Comment:
good test!

Reply:
thanks! some stuff broke with my updates so i thought it would be better to make our own files on the fly.